test/lib/actions/RulesAction.test.ts

```diff
@@ -95,3 +95,7 @@ describe('RulesAction tests', () => {
 		await expect(executionPromise).rejects.toThrow('SomeErrorFromGetAvailableEngineNames');
 	});
 });
+
+// TODO: Whenever we decide to document the custom_engine_plugin_modules flag in our configuration file, then we'll want
```

Comment:
Feel free to add in some tests now if you want... but since this is just for internal use only - I don't think we need to do this right away.